components/micro-gateway-core/src/main/ballerina/src/gateway/utils/utils.bal

```diff
@@ -905,8 +912,6 @@ function readMultipleJWTIssuers() {
     if (jwtIssuers is map<anydata>[] && jwtIssuers.length() > 0) {
         initiateJwtMap();
         printDebug(KEY_UTILS, "Found new multiple JWT issuer configs");
-        string trustStorePath = getConfigValue(LISTENER_CONF_INSTANCE_ID, TRUST_STORE_PATH, DEFAULT_TRUST_STORE_PATH);
-        string trustStorePassword = getConfigValue(LISTENER_CONF_INSTANCE_ID, TRUST_STORE_PASSWORD, DEFAULT_TRUST_STORE_PASSWORD);
```

Comment:
Isn't these values are used below

Reply:
I put this value above( calling for globally)